modules/core/src/testFixtures/java/org/apache/ignite/internal/testframework/WorkDirectoryExtension.java

```diff
@@ -56,8 +56,8 @@
  * to the module, where the tests are being run, and their paths depends on the lifecycle of the folder:
  *
  * <ol>
- *     <li>For test methods: "target/work/{@literal <name-of-the-test-class>/<name-of-the-test-method>_<current_time_millis>}"</li>
- *     <li>For test classes: "target/work/{@literal <name-of-the-test-class>/static_<current_time_millis>}"</li>
+ *     <li>For test methods: "build/work/{@literal <name-of-the-test-class>/<name-of-the-test-method>_<current_time_millis>}"</li>
```

Comment:
Seems like it can be `target` if there's `target` folder

Reply:
Gradle is going to be the main build system, so no need to explicitly mention Maven in the docs